### PR TITLE
Update gz-msgs11 badge URLs

### PR DESCRIPTION
**Pull request by caguero**
Part of https://github.com/gazebo-tooling/release-tools/issues/1092

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
 ## Gazebo Msgs 11.x
 
-### Gazebo Msgs 11.x.x (202X-XX-XX)
+### Gazebo Msgs 11.0.0 (2024-08-26)
+
+Changes since 13.4.0:
```
Comment (azeey) [inline Changelog.md:5]:
```suggestion
Changes since 10.2.0:
```

File: Changelog.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+
+1. Improve migration note on SphericalCoordinates
+    * [Pull request #452](https://github.com/gazebosim/gz-msgs/pull/452)
+
+1. Deprecate LOCAL2 in SphericalCoordinates
+    * [Pull request #451](https://github.com/gazebosim/gz-msgs/pull/451)
+
+1. Backport: Adding cone primitives.
```
Comment (azeey) [inline Changelog.md:13]:
```suggestion
1. Adding cone primitives.
```

File: Changelog.md [hunk 3]
```diff
@@ -1,6 +1,42 @@
...
-### Gazebo Msgs 11.x.x (202X-XX-XX)
+### Gazebo Msgs 11.0.0 (2024-08-26)
+
+Changes since 13.4.0:
+
+1. Improve migration note on SphericalCoordinates
+    * [Pull request #452](https://github.com/gazebosim/gz-msgs/pull/452)
+
+1. Deprecate LOCAL2 in SphericalCoordinates
+    * [Pull request #451](https://github.com/gazebosim/gz-msgs/pull/451)
```
Comment (azeey) [inline Changelog.md:11]: Consolidate

File: Changelog.md [hunk 4]
```diff
@@ -1,6 +1,42 @@
 ## Gazebo Msgs 11.x
 
-### Gazebo Msgs 11.x.x (202X-XX-XX)
+### Gazebo Msgs 11.0.0 (2024-08-26)
```
Comment (azeey) [inline Changelog.md:3]:
```suggestion
### Gazebo Msgs 11.0.0 (2024-09-xx)
```

File: README.md [hunk 5]
```diff
@@ -10,9 +10,9 @@
 Build | Status
 -- | --
 Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-msgs/tree/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-msgs/tree/main)
```
Comment (azeey) [inline README.md:12]: Can you change the branch in Test coverage as well?

File: Changelog.md (hunk 3 above)
Comment (caguero) [reply Changelog.md]: Grouped in [5832a2d](https://github.com/gazebosim/gz-msgs/pull/455/commits/5832a2d4a599e8e4dcd87e4093350fb4e0866ee2)

File: README.md (hunk 5 above)
Comment (caguero) [reply README.md]: Changed in [37f4463](https://github.com/gazebosim/gz-msgs/pull/455/commits/37f44633e73855671c0ad5928cf8172777094df4).

File: Changelog.md (hunk 1 above)
Comment (azeey) [reply Changelog.md]: Maybe, I just based it on the last version in the Changelog. I didn't check if there is a 10.3.0.

**Review by iche033 — APPROVED** [review]

**Pull request merged**